assets/src/scripts/sentry.js

```diff
@@ -0,0 +1,18 @@
+import * as Sentry from '@sentry/browser';
+import { Integrations } from '@sentry/tracing';
+
+/* global SentryParams */
+Sentry.init( {
+	dsn: SentryParams.dsn,
+	integrations: [ new Integrations.BrowserTracing() ],
+	tracesSampleRate: 1.0,
```

Comment:
curious how the sampling rate will affect server performance. Will observe during testing in staging server.

Reply:
I suggest do it dynamically, so it can be configured different for each server. An new ENV variable could solve it, since it will depend on server hardware, bandwidth and other resources.